test/jdk/java/awt/print/PrinterJob/PrintAllFonts.java

```diff
@@ -24,6 +24,7 @@
 /**
  *
  * @bug 4884389 7183516
```

Comment:
Ah, I see, it falls into the same category as all the tests in [JDK-8320676](https://bugs.openjdk.org/browse/JDK-8320676) which use the `Sysout` framework. I didn't include it because it doesn't have `@test` and therefore it can't be run.

I added a comment about the test, it's not worth filing its own bug at this point.